test/integration/user_can_place_an_order_test.rb

```diff
@@ -0,0 +1,37 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.